NGINXaaS: Updates to Overview and Prerequisites sections (#1619)
nginx/documentation: content/nginxaas-google/overview.md

## Supported regions

{{< include "/nginxaas-google/supported-regions.md" >}}

### Networking

We use Google [Private Service Connect]((https://cloud.google.com/vpc/docs/private-service-connect)) (PSC) to securely connect NGINXaaS to your applications and enable client access to your deployments. A [PSC backend](https://cloud.google.com/vpc/docs/private-service-connect#backends) brings the NGINXaaS deployment into your client network, allowing your application clients to connect seamlessly. A [PSC Interface](https://cloud.google.com/vpc/docs/private-service-connect#interfaces) brings the deployment into your application network, enabling secure connectivity to your applications. This approach gives you full control over traffic flow by leveraging your own networking resources, so you can apply your preferred security controls and ensure a secure deployment environment.

## Current Limitations

> At present, there are some constraints to be aware of regarding NGINXaaS for Google Cloud. We are actively working to enhance the offering, and many of these constraints may evolve over time as we expand and enhance our service.

Reviewer: I like the changes you made but usually just listing these limitations with no further guidance to users is also not as good of an experience.

Maybe indicate that in case a user is interested in any of these features, that they should open up a support request with us? or reach out to us in a certain way so we know.

> - At this time, we support a limited number of geographies and regions. However, we are continually working to expand support across additional locations.

Reviewer: nit:

```suggestion
- At this time, we support a limited number of geographies and regions. However, we are continually working to expand support across additional regions.
```

- We only support authentication via Google acting as an identity provider.
- User Role-Based Access Control (RBAC) is not yet supported, but this enhancement is on our roadmap as we improve access control for multi-user environments.

> - Deployments require specific NGINX configuration snippets to ensure a seamless experience. See [NGINX configuration required content]({{< ref "nginxaas-google/getting-started/nginx-configuration/overview.md#nginx-configuration-required-content" >}}) for specific details.

Reviewer 2: If configuration fixups is complete, we should be able to move away from this.. (Not a concern for this PR, but a reminder that we need to complete the necessary clean-up to close out that work.)

{{< include "/nginxaas-google/supported-regions.md" >}}

### Networking

We use Google [Private Service Connect]((https://cloud.google.com/vpc/docs/private-service-connect)) (PSC) to securely connect NGINXaaS to your applications and enable client access to your deployments. A [PSC backend](https://cloud.google.com/vpc/docs/private-service-connect#backends) brings the NGINXaaS deployment into your client network, allowing your application clients to connect seamlessly. A [PSC Interface](https://cloud.google.com/vpc/docs/private-service-connect#interfaces) brings the deployment into your application network, enabling secure connectivity to your applications. This approach gives you full control over traffic flow by leveraging your own networking resources, so you can apply your preferred security controls and ensure a secure deployment environment.

## Current Limitations

We are committed to enhancing NGINXaaS for Google Cloud and welcome your feedback to help shape the future of our service. If there are features you'd like to see prioritized, we encourage you to submit a [support ticket]({{< ref "/nginxaas-google/get-help/support.md" >}}) to share your suggestions.

> Here are the current constraints to keep in mind while using NGINXaaS for Google Cloud:

Reviewer 3: ```suggestion
Here are the current constraints you should be aware of when using NGINXaaS for Google Cloud:
```

> - At this time, we support a limited number of geographies and regions. However, we are continually working to expand support across additional regions.

Reviewer 3: ```suggestion
- NGINXaaS is [supported in a limited number of regions]({{<ref "/nginxaas/google/overview.md#supported-regions" >}}), we are continually working to expand support across additional regions.
```

Given that we say "Current" in the section header and the introduction to this list, "at this time" feels very redundant.
